core/pom.xml

```diff
@@ -224,6 +224,10 @@
       <groupId>org.scala-lang</groupId>
       <artifactId>scala-library</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.scala-lang</groupId>
+      <artifactId>scala-reflect</artifactId>
```

Comment:
?

Reply:
Jackson 2.9.8 is not dependent on scala-reflect (but 2.9.6 have this dependency), but Spark core needs this dependency, so we should add it explicitly.

Reply:
I confirmed it.